modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,26 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Connected Apps feature provides a framework that enables an external application to integrate with Anypoint Platform using https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[APIs] via standard protocols, such as SAML, OAuth 2.0, and OpenID Connect.
+Connected apps help users delegate their access without sharing sensitive credentials or giving full control of their accounts to third parties.
+Actions taken by connected apps are auditable, and users can also revoke access at any time.
+The Connected Apps feature enables you to use secure authentication protocols and control an app's access to user data. Additionally, end users can authorize the app to access their Anypoint Platform data.
```

Comment:
In the first line we are directly addressing the audience (good), but in the last line we use "end users" Who are end users? 
Is it You can control an app's access to user data and also authorize the app to access Anypoint Platform data?

Reply:
End users are non-devs and non-org admins. Regular ol' users who have Anypoint Platform credentials that they want to hand over to an app.